backend/engine/tests/test_ghas_secrets.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+from engine.plugins.ghas_secrets import main
+
+
+class TestGHASSecrets(unittest.TestCase):
+    def test_normalize_secret_type(self):
+        test_cases = [
+            ("aws_access_key_id", "aws"),
+            ("not_aws_key", "not_aws_key"),
+            ("google_api_key", "google"),
+            ("github_ssh_private_key", "ssh"),
+            ("slack_incoming_webhook_url", "slack"),
+            ("slack_workflow_webhook_url", "slack"),
```

Comment:
might want to test the fall-through case too

Reply:
I did. The second test case (line 10) is the fall-through case: `("not_aws_key", "not_aws_key")`